src/client/pages/OfferNew/Checkout/InsuranceSummaryDetails.tsx

```diff
@@ -208,10 +214,17 @@ const getHouseholdSizeValue = (householdSize: number, textKeys: TextKeyMap) => {
   )
 }
 
-const getPersonalDetails = (
-  person: OfferPersonInfo,
-  textKeys: TextKeyMap,
-): DetailsGroup => {
+type GetPersonalDetailsParams = {
+  person: OfferPersonInfo
+  textKeys: TextKeyMap
+  currentLocale: string
```

Comment:
Should this be typed as `LocaleLabel`?

Reply:
Probably, as I wrote in another comment this is kind of in-between ways to do this. I'm unsure if it's worth is messing too much with this in this PR 🤔 But I _really_ appreciate you commenting on this sub-optimal typing situation! I completely agree that it _shouldn't_ look like this